concursus-domain/src/main/java/com/opencredo/concursus/domain/commands/CommandResult.java

```diff
@@ -24,7 +24,7 @@ static CommandResult ofSuccess(UUID processingId, Instant processedTimestamp, Ty
         checkNotNull(resultType, "resultType must not be null");
         checkNotNull(resultValue, "resultValue must not be null");
 
-        if (resultType.equals(Void.class)) {
+        if (resultType.equals(void.class)) {
```

Comment:
It does - void.class is the class of the void primitive type (unlike Void.class, which is the class of the boxed Void type).